app/policies/claim_policy.rb

```diff
@@ -11,6 +11,10 @@ def confirm?
     true
   end
 
+  def destroy?
```

Comment:
Based on this PR https://github.com/DFE-Digital/itt-mentor-services/pull/391 that introduces policies, can we be a bit more strict here?

https://github.com/DFE-Digital/itt-mentor-services/blob/af7b37e6b1e39d9a72f8122423afcd55d3d2158c/app/policies/claim_policy.rb

I think only a support user can remove a claim.

Can we do something like?
```suggestion
  def destroy?
    user.support_user?
  end
  
  def remove?
    destroy?
  end
```